client/roundCtrl.ts

```diff
@@ -163,6 +116,8 @@ export default class RoundController {
                           // If server received and processed it the first time, it will just ignore it
 
     constructor(el: HTMLElement, model: PyChessModel) {
+        super(el, model);
+        this.materialFunction = updateMaterial;

```

Comment:
I don't think an explicit `materialFunction` class member is necessary. Just call `updateMaterial(ctrl)` when appropriate.

Reply:
Done.